### DISCUSSION TITLE
[Mobile] Caching Network Images

### DISCUSSION THREAD

**Discussion by apoleo88** [singerdmx/flutter-quill#827] [Mobile]
Testing on Android, on the last version of the stable channel.

After an Image is shown once, it seems to cache it, and next time it will use the cached version. 

But if the app is closed and reopened it will retrieve it again the first time and if there is no connection it won't show it.

I would like instead to tell him to use the cached version, in order to work offline and save network data. (maybe check if it is changed once in a while)

Is it possible?

**Comment by the-thirteenth-fox** [comment]
+1

**Comment by singerdmx** [comment]
This is regarding embed builder.
<img width="1005" alt="image" src="https://user-images.githubusercontent.com/122956/180941360-162ea7a6-6eb0-43c9-a34e-29db60a5722d.png">


You can take a look and submit a PR
